File: src/Iniciante/1015.java
package Iniciante;

import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner input = new Scanner(System.in);
        double x1 = input.nextDouble();
        double y1 = input.nextDouble();
        double x2 = input.nextDouble();
        double y2 = input.nextDouble();

        double xx = x2-x1;
        double yy = y2-y1;
        double d = Math.sqrt(Math.pow(xx, 2)+Math.pow(yy, 2));
        String resultado = String.format("%.4f", d);
        System.out.println(resultado);


    }    
}

File: src/Iniciante/1015.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>
#include <math.h>

float distancia(float a, float b, float c, float e)
{

    float dd;
    dd = sqrt(pow((c - a), 2) + pow((e - b), 2));
    return (dd);
}

int main()
{
    setlocale(LC_ALL, "pt-br");
    float x1, y1, x2, y2, dd;

    scanf("%f %f", &x1, &y1);
    scanf("%f %f", &x2, &y2);

    dd = distancia(x1, y1, x2, y2);
    printf("%.4f\n", dd);
    return 0;
}